libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/EHorizonPosition.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.base.trip.model
+
+/**
+ * The position on the current EHorizon.
+ */
+data class EHorizonPosition(
+    /**
+     * The current Edge.
+     */
+    // TODO Check EHorizonPosition included the Edge and not the edgeId in the original design
+    // var edge: Edge,
+    var edgeId: Int,
+
+    /**
+     * The progress along the current edge [0,1)
+     */
+    var t: Double
```

Comment:
Curious why progress is called t, why not p ?

Reply:
Took the nomenclature from our original draft design. BTW NN's `ElectronicHorizonPosition` is also using `t` 👀 

```java
public final class ElectronicHorizonPosition implements Serializable {
    private final long edgeId;
    private final double t;
```

@mskurydin @SiarheiFedartsou do you remember why `t` instead of `p`?

Reply:
What about percentAlong?

Reply:
Internally will change for GraphPosition. On SDK side: ElectronicHorizonPosition wrapping GraphPosition.